<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeachersForm_template</class>
 <widget class="QDialog" name="TeachersForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="Title of dialog">Teachers</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="3">
        <widget class="QListWidget" name="teachersListWidget">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="activateAllPushButton">
           <property name="text">
            <string>Activate all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deactivateAllPushButton">
           <property name="text">
            <string>Deactivate all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveTeacherUpPushButton">
           <property name="text">
            <string comment="It means move the item up in the list">Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveTeacherDownPushButton">
           <property name="text">
            <string comment="It means move the item down in the list">Down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sortTeachersPushButton">
           <property name="text">
            <string>Sort</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Current teacher</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QPlainTextEdit" name="currentTeacherTextEdit">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="renameTeacherPushButton">
           <property name="text">
            <string>Rename</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="activateTeacherPushButton">
           <property name="text">
            <string>Activate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deactivateTeacherPushButton">
           <property name="text">
            <string>Deactivate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qualifiedSubjectsPushButton">
           <property name="text">
            <string comment="It means a list of qualified (allowed) subjects for the teacher. Keep translation short.">Qualified subjects</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="targetNumberOfHoursPushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string comment="It means the target number of hours for a teacher, but keep translation short">Target hours</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="commentsPushButton">
           <property name="text">
            <string>Comments</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeTeacherPushButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addTeacherPushButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closePushButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
